### ISSUE TITLE
Submenus stay opened and may obscure focused elements below (WCAG 2.2 - 2.4.11 + 2.4.12 Focus Not Obscured)

### ISSUE THREAD

**Issue by PixelBaer** [joomla/joomla-cms#45917]
### Steps to reproduce the issue

Desktop Issue:
1. On Desktop Open Joomla demosite https://cassiopeia.joomla.com/sample-layouts/articles/millions
2. Open Sample Layouts Submenu 
3. Use Tab key to navigate to the Breadcrumbs
4. The Link "Articles" gets focussed yet is completely obscured by the submenu (or partially, depending on zoom, etc) 

Mobile Issue:
1. On Mobile (or with resized window) Open Joomla demosite https://cassiopeia.joomla.com/sample-layouts/articles/millions
2. Open the Styles submenu
3. Use Tab to navigate to search
4. Search input and button are obscured, even though they have focus

### Expected result
The focussed element should be visible

### Actual result
The focused element is obscured by the submenu flyout

### Device and system info

* What version of Joomla were you using?
5.3.2

* What device were you using? (eg, phone, laptop)
Laptop

* What web browser were you using? (eg, Chrome, Safari)
Chrome / MacOs

* If you were using assistive technology, please let us know the device or software you were using (eg VoiceOver, JAWS, sip-and-puff device, etc)

### Additional comments

If you know which [WCAG](https://www.w3.org/TR/WCAG/) or [ATAG](https://www.w3.org/TR/ATAG20/) criteria this violates, please indicate it here:
This behavior violates WCAG 2.2 2.4.11 and potentially 2.4.12
https://www.w3.org/TR/WCAG22/#focus-not-obscured-minimum

**Comment by chmst** [comment]
Maybe resolved in https://github.com/joomla/joomla-cms/pull/45922?
